Bottom: 3a88ac0deae03760a3e66ffdd5de6e4ad7e2e45b
Top:    2287b53ad44e1ff759f1bc5d3ecdba0e4b7e8a8a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2017-02-26 18:03:30 +0000

Add comment to VIA_IS_BURIED


---

diff --git a/src/macro.h b/src/macro.h
index aeccd7e..f376f5d 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -483,5 +483,5 @@ extern int mem_any_set (unsigned char *, int);
 
 #endif
 
-#define VIA_IS_BURIED(via) (via->BuriedFrom != 0 || via->BuriedTo != 0)
+#define VIA_IS_BURIED(via) (via->BuriedFrom != 0 || via->BuriedTo != 0) /*!< Despite the naming, this identifies any blind or buried via (any via which does not go right through the PCB) */
 #define VIA_ON_LAYER(via, layer) (layer >= via->BuriedFrom && layer <= via->BuriedTo )
